certified-connectors/GreenSign/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "GreenSign",
+          "clientId": "<<Please add your clientID here>>",
+          "scopes": [
+            "default"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {

```

Comment:
Kindly change redirectMode to GlobalPerConnector, please refer the below link and modify duly.

Reply:
I have made the requested change. Please re-review.

Reply:
Thank you very much for the changes requested.  Could you please remove the redirectUrl. Please refer the below link. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4e-validate-your-custom-connector-files

Reply:
Removed.